Jenkinsfile.integration

```diff
@@ -128,16 +128,10 @@ pipeline {
                 timeout(time: 45, unit: 'MINUTES', activity: true)
             }
             parallel {
-                stage('Deploy CaaSP3') {
-                    when { expression  { params.caasp_version == "caasp3" } }
-                    steps {
-                        sh "./run.sh deploy_caasp"
-                    }
-                }
                 stage('Deploy CaaSP4') {
                     when { expression  { params.caasp_version == "caasp4" } }
```

Comment:
remove if caasp4 only as it should alwasy run

Reply:
Yes, good point